hosts/binarycache/configuration.nix

```diff
@@ -54,13 +54,18 @@
   };
 
   services.nginx = {
+    recommendedZstdSettings = true;
     virtualHosts = {
       "cache.vedenemo.dev" = {
         enableACME = true;
         forceSSL = true;
         default = true;
         locations."/" = {
           proxyPass = "http://${config.services.nix-serve.bindAddress}:${toString config.services.nix-serve.port}";
+          extraConfig = ''
+            zstd on;
+            zstd_types application/x-nix-archive;
+          '';
```

Comment:
This still needs `proxyWebsockets = true;` inside the location block to match the previous config.

Reply:
Nix doesn't know anything about websockets, we don't need to add that support.

Reply:
At least the harmonia readme example sets those config options. But it's true they might be unnecessary.

Reply:
I'll leave it out for the moment.